src/core_modules/capture-core/components/PossibleDuplicatesDialog/possibleDuplicatesDialog.epics.js

```diff
@@ -0,0 +1,90 @@
+// @flow
```

Comment:
this is also the deleted file `SearchGroupDuplicate/searchGroupDuplicate.epics.js` Its been a whle that I havent cheched this PR but I believe I havent made any changes in this file.